reference-implementation/test/transform-stream.js

```diff
@@ -367,3 +367,25 @@ test('TransformStream flush gets a chance to enqueue more into the readable, and
   })
   .catch(e => t.error(e));
 });
+
+test('Transform context', t => {
```

Comment:
Test name should be "Transform stream should call transformer methods as methods"